src/App.tsx

```diff
@@ -1,223 +1,220 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+  FormEvent,
+} from 'react';
+import classnames from 'classnames';
+import {
+  createTodo,
+  getTodos,
+  removeTodo,
+  updateTodo,
+} from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Footer } from './components/Footer';
+import { TodosList } from './components/TodosList';
+import { FilterType } from './types/FilterStatus';
+import { Todo } from './types/Todo';
+import { Error } from './types/Error';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [error, setError] = useState<Error | null>(null);
+  const [filterType, setFilterType] = useState('all');
```

Comment:
```suggestion
  const [filterType, setFilterType] = useState(FilterState.All);
```

Not every string can be a filter value